aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -333,30 +357,45 @@ module aptos_framework::account {
         cap_update_table: vector<u8>,
     ) acquires Account, OriginatingAddress {
         let addr = signer::address_of(account);
-        assert!(exists_at(addr), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+        assert!(
+            exists_at(addr),
+            error::not_found(EACCOUNT_DOES_NOT_EXIST)
+        );
         let account_resource = borrow_global_mut<Account>(addr);
 
         // Verify the given `from_public_key_bytes` matches this account's current authentication key.
         if (from_scheme == ED25519_SCHEME) {
-            let from_pk = ed25519::new_unvalidated_public_key_from_bytes(from_public_key_bytes);
-            let from_auth_key = ed25519::unvalidated_public_key_to_authentication_key(&from_pk);
+            let from_pk = ed25519::new_unvalidated_public_key_from_bytes(
+                from_public_key_bytes
+            );
+            let from_auth_key = ed25519::unvalidated_public_key_to_authentication_key(
+                &from_pk
+            );
             assert!(
                 account_resource.authentication_key == from_auth_key,
                 error::unauthenticated(EWRONG_CURRENT_PUBLIC_KEY)
             );
-        } else if (from_scheme == MULTI_ED25519_SCHEME) {
-            let from_pk = multi_ed25519::new_unvalidated_public_key_from_bytes(from_public_key_bytes);
-            let from_auth_key = multi_ed25519::unvalidated_public_key_to_authentication_key(&from_pk);
+        }
+        else if (from_scheme == MULTI_ED25519_SCHEME) {
+            let from_pk = multi_ed25519::new_unvalidated_public_key_from_bytes(
+                from_public_key_bytes
+            );
+            let from_auth_key = multi_ed25519::unvalidated_public_key_to_authentication_key(
+                &from_pk
+            );
             assert!(
                 account_resource.authentication_key == from_auth_key,
                 error::unauthenticated(EWRONG_CURRENT_PUBLIC_KEY)
             );
-        } else {
+        }
+        else {
```

Comment:
The position of this `else` is off

Reply:
fixed at https://github.com/movebit/movefmt/commit/64839f36df83bd31864e1cb70ac38e17ef645303